src/containers/Marker/container.js

```diff
@@ -1,14 +1,15 @@
 import { connect } from 'react-redux';
-import * as actions from '../../redux/actions/index';
 import { Marker } from 'google-maps-react';
 
-const mapStateToProps = (state) => ({
+import { mapActions } from '../../redux/actions';
+
+const mapStateToProps = ({ mapConfig: { coords }, mapInfo: { events: { markerOnClick } } }) => ({
   name: 'Your position',
-  position: state.options.coords,
-  onClick: state.options.events.markerOnClick
+  position: coords,
+  onClick: markerOnClick,
```

Comment:
position -> coords, onClick -> markerOnClick. What's the use of renaming properties? Can they be named the needed way from the beginning?